client/tso_stream.go

```diff
@@ -141,6 +141,7 @@ func (s *pdTSOStream) processRequests(
 	}
 	tsoBatchSendLatency.Observe(time.Since(batchStartTime).Seconds())
 	resp, err := s.stream.Recv()
+	requestDurationTSO.Observe(time.Since(start).Seconds())
```

Comment:
Do we need to distinguish them?

Reply:
Do you mean to distinguish the duration between success and failure? Currently, I don't think it's necessary because we generally only use this metric to compare `TSO Wait Duration` to infer the source of the delay, the latter does not distinguish between successful and failed requests.

Reply:
Yes, I was wondering if we find a latency spike, we might need to know if the request has succeeded or failed.

Reply:
Make sense. Addressed. PTAL.